zerver/openapi/zulip.yaml

```diff
@@ -7189,11 +7192,20 @@ paths:
                         example: 1
                       delivery_email:
                         type: string
+                        nullable: true
                         description: |
-                          The user's real email address. This field is present only if
-                          [email address visibility](/help/restrict-visibility-of-email-addresses) is
-                          limited and you are an administrator with access to real email addresses
-                          under the configured policy.
+                          The user's real email address. This value will be `None` if you cannot
+                          access this user's real email address. For bot users, this field is
+                          always set to the real email of the bot, because bot users always have
+                          `email_address_visibility` as `EMAIL_ADDRESS_VISIBILITY_EVERYONE`.
```

Comment:
commit-structure nit: there's a few changes in this commit:

588c45f3b settings: Add backend code for using user email_address_visibility setting.

that look like they were meant to be squashed into the previous one. (They're wording changes to the API docs, but this commit doesn't otherwise touch the API.)

Reply:
Yeah, so the change to always show emails for bots is done in the previous commit `users: Always pass "delivery_email" field in user objects` but as Tim suggested [here](https://github.com/zulip/zulip/pull/22994#discussion_r1099493434) to write it in different way and we actually use the user-level setting in commit ` settings: Add backend code for using user email_address_visibility setting.`  I am not sure whether we are fine with such type of changes but thought it would be misleading to mention about the user-level setting when we are actually not using it.